assets/js/googlesitekit/api/cache.js

```diff
@@ -29,7 +36,7 @@ import { HOUR_IN_SECONDS } from '../../util';
  * @since 1.5.0
```

Comment:
How about we update this with an annotation about this change since it's a rather significant one?

```suggestion
 * @since 1.5.0
 * @since n.e.x.t Updated to include a user, session, and blog-specific hash.
```